internal/config/config.go

```diff
@@ -108,58 +167,7 @@ func Reset() {
 }
 
 func read() Config {
-	config := Config{}
-
-	applyEnvironmentConfiguration := func(config Config) Config {
-		ryukDisabledEnv := os.Getenv("TESTCONTAINERS_RYUK_DISABLED")
-		if parseBool(ryukDisabledEnv) {
-			config.RyukDisabled = ryukDisabledEnv == "true"
-		}
-
-		hubImageNamePrefix := os.Getenv("TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX")
-		if hubImageNamePrefix != "" {
-			config.HubImageNamePrefix = hubImageNamePrefix
-		}
-
-		ryukPrivilegedEnv := os.Getenv("TESTCONTAINERS_RYUK_CONTAINER_PRIVILEGED")
-		if parseBool(ryukPrivilegedEnv) {
-			config.RyukPrivileged = ryukPrivilegedEnv == "true"
-		}
-
-		ryukVerboseEnv := readTestcontainersEnv("RYUK_VERBOSE")
-		if parseBool(ryukVerboseEnv) {
-			config.RyukVerbose = ryukVerboseEnv == "true"
-		}
-
-		ryukReconnectionTimeoutEnv := readTestcontainersEnv("RYUK_RECONNECTION_TIMEOUT")
-		if timeout, err := time.ParseDuration(ryukReconnectionTimeoutEnv); err == nil {
-			config.RyukReconnectionTimeout = timeout
-		}
-
-		ryukConnectionTimeoutEnv := readTestcontainersEnv("RYUK_CONNECTION_TIMEOUT")
-		if timeout, err := time.ParseDuration(ryukConnectionTimeoutEnv); err == nil {
-			config.RyukConnectionTimeout = timeout
-		}
-
-		return config
-	}
-
-	home, err := os.UserHomeDir()
-	if err != nil {
-		return applyEnvironmentConfiguration(config)
-	}
-
-	tcProp := filepath.Join(home, ".testcontainers.properties")
-	// init from a file
-	properties, err := properties.LoadFile(tcProp, properties.UTF8)
-	if err != nil {
-		return applyEnvironmentConfiguration(config)
-	}
-
-	if err := properties.Decode(&config); err != nil {
-		fmt.Printf("invalid testcontainers properties file, returning an empty Testcontainers configuration: %v\n", err)
-		return applyEnvironmentConfiguration(config)
-	}
+	config := defaultConfig()
 
 	return applyEnvironmentConfiguration(config)
```

Comment:
bug: this is already applied

Reply:
Following the code, if no if-branch is eval to true, then the config won't receive the env config. I think we could move the apply call to the defaultConfig function, so that it only happens there.